.github/workflows/parachain.yml

```
@@ -15,6 +15,11 @@ on:
       - "!parachain/LICENSE"
   workflow_dispatch:
 
+env:
+  FUZZ_MAX_LEN: 10000000000
+  FUZZ_MAX_RUNS: 1000000
+  RUST_NIGHTLY: "2023-06-14"
```

We're sticking to `2023-05-23` for now, as [Ron noticed in a previous PR](https://github.com/Snowfork/snowbridge/pull/918#discussion_r1281916653) that that's the version Parity appears to use.
```suggestion
  RUST_NIGHTLY: "2023-05-23"
```

---

Addressed in https://github.com/Snowfork/snowbridge/pull/907/commits/eeb231d9d7a850c4fa772a32b2cdbb24a153cd98.